mitmproxy/exceptions.py

```diff
@@ -112,6 +112,10 @@ class AddonHalt(MitmproxyException):
     pass
 
 
+class TypeError(MitmproxyException):
```

Comment:
This looks wrong - we should use a regular Python TypeError?

Reply:
This is an unfortunate clash of names, but I'd like to keep this type separate from the built-in TypeError.